stdlib/builtins.pyi

```diff
@@ -1162,7 +1165,7 @@ class frozenset(AbstractSet[_T_co]):
     @overload
     def __new__(cls) -> Self: ...
     @overload
-    def __new__(cls, iterable: Iterable[_T_co], /) -> Self: ...
+    def __new__(cls, iterable: Iterable[_T_co] | _GetItemIterable[_T], /) -> Self: ...
```

Comment:
```suggestion
    def __new__(cls, iterable: Iterable[_T_co] | _GetItemIterable[_T_co], /) -> Self: ...
```
This might be causing some of the mypy-primer fallout